File: app/src/main/cpp/Node.cpp
#include "Node.h"


Node::Node()
{
}

Node::Node(int h, int w , int value)
{
	Node::h = h;
	Node::w = w;
	Node::value = value;
}

Node::~Node()
{
}

int Node::getH()
{
	return h;
}

int Node::getW()
{
	return w;
}

int Node::getValue()
{
	return value;
}

void Node::setH(int h)
{
	Node::h = h;
}

void Node::setW(int w)
{
	Node::w = w;
}

void Node::setValue(int value)
{
	Node::value = value;
}

File: hrjar/src/main/java/cn/hazeremovecamera/jar/Node.java
package cn.hazeremovecamera.jar;

/**
 * Created by OptimusV5 on 2015/1/8.
 */
public class Node {
    private int h;
    private int w;
    private int value;
    Node() {

    }
    Node(int h, int w,int value) {
        this.h = h;
        this.w = w;
        this.value = value;
    }

    public int getH() {
        return h;
    }

    public int getW() {
        return w;
    }

    public int getValue() {
        return value;
    }
    public void setH(int h) {
        this.h = h;
    }
    public void setW(int w) {
        this.w = w;
    }
    public void setValue(int value){
        this.value = value;
    }
}
